web/geo/generateZoneBoundingBoxes.ts

```diff
@@ -6,117 +6,139 @@ import { saveZoneYaml } from './files.js';
-let isAggregate = false;
-if (zoneFeatures.length <= 0) {
-  console.info(`Zone ${zoneKey} does not exist in geojson, using subzones instead`);
-  isAggregate = true;
-  zoneFeatures = zonesGeo.features.filter((d) => {
-    return d.properties.countryKey === zoneKey;
-  });
-}
-zonesGeo.features = zoneFeatures;
+  let zoneFeatures = zonesGeo.features.filter((d) => d.properties.zoneName === zoneKey);
+  let isAggregate = false;
+  if (zoneFeatures.length <= 0) {
+    console.info(`Zone ${zoneKey} does not exist in geojson, using subzones instead`);
+    isAggregate = true;
+    zoneFeatures = zonesGeo.features.filter((d) => {
+      return d.properties.countryKey === zoneKey;
+    });
+  }
 
-let allCoords: Position[] = [];
-let boundingBoxes: { [key: string]: number[][] } = {};
+  let allCoords: Position[] = [];
+  let boundingBoxes: { [key: string]: number[][] } = {};
 
-for (const zone of zonesGeo.features) {
-  allCoords = [];
-  const geometryType = zone.geometry.type;
-  for (const coords1 of zone.geometry.coordinates) {
-    for (const coord of coords1[0]) {
-      allCoords.push(coord as Position);
+  for (const zone of zoneFeatures) {
+    allCoords = [];
+    const geometryType = zone.geometry.type;
+    for (const coords1 of zone.geometry.coordinates) {
+      for (const coord of coords1[0]) {
+        allCoords.push(coord as Position);
+      }
     }
-  }
 
-  let minLat = 200;
-  let maxLat = -200;
-  let minLon = 200;
-  let maxLon = -200;
-
-  if (geometryType == 'MultiPolygon') {
-    for (const coord of allCoords) {
-      const lon = coord[0];
-      const lat = coord[1];
+    let minLat = 200;
+    let maxLat = -200;
+    let minLon = 200;
+    let maxLon = -200;
+
+    if (geometryType == 'MultiPolygon') {
+      for (const coord of allCoords) {
+        const lon = coord[0];
+        const lat = coord[1];
+
+        minLon = Math.min(minLon, lon);
+        maxLon = Math.max(maxLon, lon);
+        minLat = Math.min(minLat, lat);
+        maxLat = Math.max(maxLat, lat);
+      }
+    } else {
+      const lon = allCoords[0] as unknown as number;
+      const lat = allCoords[1] as unknown as number;
 
       minLon = Math.min(minLon, lon);
       maxLon = Math.max(maxLon, lon);
       minLat = Math.min(minLat, lat);
       maxLat = Math.max(maxLat, lat);
     }
-  } else {
-    const lon = allCoords[0] as unknown as number;
-    const lat = allCoords[1] as unknown as number;
-
-    minLon = Math.min(minLon, lon);
-    maxLon = Math.max(maxLon, lon);
-    minLat = Math.min(minLat, lat);
-    maxLat = Math.max(maxLat, lat);
+
+    boundingBoxes[zone.properties.zoneName] = [
+      [minLon - 0.5, minLat - 0.5],
+      [maxLon + 0.5, maxLat + 0.5],
+    ];
   }
 
-  boundingBoxes[zone.properties.zoneName] = [
-    [minLon - 0.5, minLat - 0.5],
-    [maxLon + 0.5, maxLat + 0.5],
-  ];
-}
+  if (isAggregate) {
+    const averageBoundingBox: number[][] = [
+      [200, 200],
+      [-200, -200],
+    ];
```

Comment:
Do we need a average bouding box or could we use a initial state of Number.NaN istead?
I think it would make more sense to start from zero so to speak.

Maybe call it temp bounding box insteade of average as well.